### PR TITLE
feat: gnap server returns granted access tokens when re-requested

### PR DESCRIPTION
**Pull request by Moopli**
Signed-off-by: Filip Burlacu <filip.burlacu@securekey.com>

### CODE REVIEW

File: pkg/gnap/accesspolicy/access_policy.go [hunk 1]
```diff
@@ -366,10 +366,10 @@ func (ap *AccessPolicy) parse(tok gnap.TokenAccess) (tokenAccessMap, error) {
 	return tokMap, nil
 }
 
-// isTokenAccessSubset returns true iff TokenAccess sub is a subset of super.
+// isTokenAccessSuperset returns true iff TokenAccess super is a superset of sub.
```
Comment (birtony) [inline pkg/gnap/accesspolicy/access_policy.go:369]:
```suggestion
// isTokenAccessSuperset returns true if TokenAccess super is a superset of sub.
```

File: pkg/gnap/api/api.go [hunk 2]
```diff
@@ -58,6 +58,11 @@ type AccessMetadata struct {
 	SubjectKeys []string
 }
 
+// IsEmpty returns true iff the AccessMetadata object is empty.
```
Comment (birtony) [inline pkg/gnap/api/api.go:61]:
```suggestion
// IsEmpty returns true if the AccessMetadata object is empty.
```

**Review by talwinder50 — APPROVED** [review]

File: pkg/gnap/api/api.go (hunk 2 above)
Comment (Moopli) [reply pkg/gnap/api/api.go:61]: iff means if and only if

File: pkg/gnap/accesspolicy/access_policy.go (hunk 1 above)
Comment (Moopli) [reply pkg/gnap/accesspolicy/access_policy.go:369]: iff means if and only if

**Pull request merged**